main.py

```diff
@@ -1 +1,28 @@
-print("DUPA BLADA")
\ No newline at end of file
+from Bike import *
+from BikeRepository import *
+from functions import *
+import pyodbc
+
+
+print("Welcome to Bike Rent.")
+userChoice = ''
+
+while userChoice != 'q':
+    print("1. Add bike")
+    print("2. Add user")
+    userChoice = input("Enter num or q to quit: ")
+    cnxn = pyodbc.connect('DRIVER={ODBC Driver 17 for SQL Server};SERVER=USER-KOMPUTER\SQLEXPRESS;DATABASE=BikeRents;Trusted_Connection=yes')
```

Comment:
za kazdym jak wejdzie w add user tworzone jest polaczenie. chcielismy miec jedno na caly cykl zycia programu. wynies to do góry